### PR TITLE
ts: assorted test refactors

### PR DESCRIPTION
**Pull request by mrtracy**
Two commits: 

+ Some small mechanical renames for clarity + some missing comments
+ Move iterator tests to their own file, separate from query tests.

### CODE REVIEW

File: pkg/ts/memory.go [hunk 1]
```diff
@@ -36,8 +36,17 @@ var (
...
 // QueryMemoryOptions represents the adjustable options of a QueryMemoryContext.
 type QueryMemoryOptions struct {
-	BudgetBytes             int64
-	EstimatedSources        int64
+	// BudgetBytes is the maximum number of bytes that should be reserved by this
+	// query at any one time.
+	BudgetBytes int64
+	// EstimatedSources is an estimate of the number of distinct sources that this
+	// query will encounter on disk. This is needed to better estimate how much
+	// memory a query will actually consumer.
```
Comment (tschottdorf) [inline pkg/ts/memory.go]: Typo

**Review by tbg — APPROVED** [review]

File: pkg/ts/memory.go (hunk 1 above)
Comment (mrtracy) [reply pkg/ts/memory.go]: Fixed

**Comment by tschottdorf** [comment]
For posterity, the first build failed because of Docker:

> docker: error pulling image configuration: Get https://registry-1.docker.io/v2/cockroachdb/builder/blobs/sha256:7a9ce7a079ccc5b8be9bdd2a6340885eebe31a79e6a586d96f9f39c5ec15972c: EOF.

**Pull request merged**